R/api.R

```diff
@@ -367,14 +368,24 @@ endpoint_adr_metadata <- function(queue) {
                                     returning = response)
 }
 
-
 endpoint_upload_input <- function(queue) {
   input <- porcelain::porcelain_input_body_binary("file")
   response <- porcelain::porcelain_returning_json(
     "SessionFile.schema", schema_root())
   porcelain::porcelain_endpoint$new("POST",
-                                    "internal/upload/input/<filename>",
-                                    upload_input(queue),
+                                    "/internal/upload/input/<filename>",
```

Comment:
Looks like we have `uploads_dir` => `upload/input` and `results_dir` => `upload/output` - want to tidy that up?

Reply:
Ok have made this a bit less confusing hopefully,

`inputs_dir` => `upload/input` and `results_dir` => `upload/result`